data/map planets.txt

```diff
@@ -1099,6 +1099,12 @@ planet "Dlia Jzaur"
 	description `	Oddly enough, Dlia Jzaur has many unnatural looking formations on the surface that seem to reach up to the sky in areas where eclipses often occur. The formations are hollow underneath, but show no signs of Gegno involvement.`
 	government Uninhabited
 
+planet "Donya"
+	attributes uninhabited
+	landscape land/fields29
```

Comment:
Is this the right image for this planet? The short grass implies human habitation.

Reply:
I was thinking (and I guess this wasn't reflected in the description) that the soil has nutrients that helped against the rain. The grasses get melted by the rain down until that specific point.

Reply:
Interesting... so the original colonists created the perfect landscaping system, but then couldn't control it? I like that idea a lot.